e2e/go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/fluxcd/source-controller/api v0.25.11
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/hashicorp/vault/api v1.9.2
+	github.com/michaelsauter/dsv-sdk-go v0.0.0-20230614073225-faaa146344bf
```

Comment:
This is temporary only. Before merging I would like to change this to `github.com/DelineaXPM/dsv-sdk-go/v2`, which needs to adopt https://github.com/DelineaXPM/dsv-sdk-go/pull/35 for the E2E tests to be feasible.

Reply:
@michaelsauter it's merged now and newly tagged release available. Good luck! We have the dsv-k8s repo that does secret syncing but I didn't have a chance to look into adding it to this project. Awesome to see you jumping in on this. cheers!

Reply:
Thanks! I've seen dsv-k8s but we would rather use a solution like external-secrets operator which supports multiple backends and likely will see more active development going forward.